service/src/main/java/uk/gov/hmcts/reform/fpl/model/event/MessageJudgeEventData.java

```diff
@@ -22,14 +23,15 @@
     JudicialMessage judicialMessageReply;
 
     public static String[] transientFields() {
-        return new String[] {
+        return new String[]{
             "hasAdditionalApplications", "isMessageRegardingAdditionalApplications",
             "additionalApplicationsDynamicList", "relatedDocumentsLabel",
             "nextHearingLabel", "judicialMessageMetaData", "judicialMessageNote", "judicialMessageDynamicList",
             "messageJudgeOption", "judicialMessageReply", "hasJudicialMessages"
         };
     }
 
+    @JsonIgnore
```

Comment:
was that a bug?

Reply:
not a bug, but there is no such filed in ccd def, basically MessageJudgeEventData is never put into caseData. 
but in a test when empty CaseData is created and converted to CaseDetails then this filed appears, which is not intended

Reply:
ok good to know. a bit counterintuitive with @JsonUnwrapped, good to have @jsonIgnore